### PR TITLE
Trustkit configuration

### PR DESCRIPTION
**Pull request by rclee**
Adds new hashes and prepares for upcoming backend configurations.

### CODE REVIEW

File: MapboxMobileEvents/MMEHashProvider.h [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
+#import "MMEEventsConfiguration.h"
+
+@interface MMEHashProvider : NSObject
+
+@property (nonatomic) NSArray *cnHashes;
+@property (nonatomic) NSArray *comHashes;
```
Comment (julianrex) [inline MapboxMobileEvents/MMEHashProvider.h]: Are these only supposed to be set by `updateHashesWithConfiguration `? If so, you might want to declare these as `readonly` in the header file, then redeclare as `readwrite` in the implementation file.

**Review by julianrex — DISMISSED** [review]
One minor question, but looks good

File: MapboxMobileEvents/MMEHashProvider.h (hunk 1 above)
Comment (rclee) [reply MapboxMobileEvents/MMEHashProvider.h]: Yes! Good call

**Comment by rclee** [comment]
Closing: Merged back with https://github.com/mapbox/mapbox-events-ios/issues/56

**Pull request closed without merging**